packages/liveblocks-core/src/__tests__/room.test.ts

```diff
@@ -58,7 +58,7 @@ const defaultRoomConfig: RoomConfig = {
   roomId: "room-id",
   throttleDelay: THROTTLE_DELAY,
   lostConnectionTimeout: 99999,
-  liveblocksServer: "wss://live.liveblocks.io/v7",
+  baseUrl: "https://api.liveblocks.io",
```

Comment:
Should this use the same DEFAULT_BASE_URL from constants as __tests__/_utils.ts?

Reply:
Yeah, looks like I missed it — thanks for catching that 👍